docs/UserGuide.md

```diff
@@ -296,9 +309,12 @@ Format: `schedule add candidate/INDEX at/DATE_TIME`
 * The candidate index must be a positive integer 1, 2, 3, …​
 * `DATE_TIME` must be specified in the format `dd-MM-yyyy HH:mm`.
 * `DATE_TIME` must not be earlier than the present date and time.
-* Interview duration is fixed at 30 minutes. Attempts to schedule an interview within the duration of another interview will
-result in an error. (e.g. Interview A starts at 10AM on a given day. Attempts to schedule an interview from 9:31AM up to 10:29AM is prohibited.)
+* If the day `dd` provided in `DATE_TIME` is not compatible with the month `MM` and year `yyyy`, the nearest valid date will be matched.
```

Comment:
hmmm dyou think we would be able to verify the validity of the date input in our logic instead? I'm not sure if this will still be flagged out as a functionality bug even if its the java api :O

Reply:
oh dear @lzan98 linking this here https://github.com/nus-cs2103-AY2122S2/forum/issues/260#issuecomment-1091019905

don't think this is acceptable as a functionality bug. sorry then I guess we can leave this part as it is :)